packages/authenticator/amplify_authenticator/example/macos/Flutter/GeneratedPluginRegistrant.swift

```diff
@@ -18,7 +18,7 @@ func RegisterGeneratedPlugins(registry: FlutterPluginRegistry) {
   AmplifySecureStoragePlugin.register(with: registry.registrar(forPlugin: "AmplifySecureStoragePlugin"))
   ConnectivityPlugin.register(with: registry.registrar(forPlugin: "ConnectivityPlugin"))
   DeviceInfoPlusMacosPlugin.register(with: registry.registrar(forPlugin: "DeviceInfoPlusMacosPlugin"))
-  FLTPackageInfoPlusPlugin.register(with: registry.registrar(forPlugin: "FLTPackageInfoPlusPlugin"))
+  FPPPackageInfoPlusPlugin.register(with: registry.registrar(forPlugin: "FPPPackageInfoPlusPlugin"))
```

Comment:
*Question:* Should this be included?

It is part of an example app. If it is needed for tests to run it should probably be merged to main and then cherry picked (to avoid future merge conflicts). If it isn't needed for tests, I would remove this as it isn't part of the source code.